WIP: readme restructure (#32)
openshift/csi-driver-projected-resource: docs/deep-dive.md

# Some more detail and examples around our vetted scenarios

> ## What happens exactly if the Share is not there when you create a Pod that references it

Reviewer: Grammar nit:
```suggestion
## What happens if the Share does not exist when you create a Pod that references it?
```
